src/app/core/session/login/login.component.scss

```diff
@@ -29,3 +32,7 @@
   color: red;
   font-weight: bold;
 }
+
+.password-icon {
+  color: mat.get-color-from-palette($ndb-app-primary);
```

Comment:
I think it would look cleaner if the color would be the same as the other elements in the form. So something black-/gray-ish

Reply:
Yes, of course; I'm very fine with that.